exercises/numbers/remainderOf/remainderOf.js

```diff
@@ -16,6 +16,10 @@
  * @returns {boolean} True if num is even and false otherwise
  */
 function remainderOf(n, d) {
+  while (n >= d){
```

Comment:
You can use `%`, although the recursion works. That said, the `while` statement here doesn't make sense as above. We either enter the loop body or we don't. If we do, we `return` immediately. So the loop body is run either one time or zero times.

In other words, it's functionally acting like an `if` statement.

You could do:

```js
function remainderOf(n, d) {
  if (n < d) {
    return n;
  }

  return remainderOf(n - d, d);
}
```

But as with any recursive solution, there's an upper limit on the size of the inputs. Do it iteratively unless:

1. You're practicing recursion
2. There's a theoretical or practical upper-limit on the size of the inputs